omnibus/resources/agent/msi/cal/CustomAction.cpp

```diff
@@ -164,37 +165,43 @@ extern "C" UINT __stdcall FinalizeInstall(MSIHANDLE hInstall) {
             keyInstall.setStringValue(installCreatedDDDomain.c_str(), data.getDomainPtr());
         }
     }
+    
+    // since we just created the user, fix up all the rights we want
```

Comment:
```suggestion
    // add all the rights we want to the user (either existing or newly created)
```
Comment no longer true.